src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/AuthenticationSuccessHandler.php

```diff
@@ -37,8 +37,8 @@ public function __construct(
 
     public function onAuthenticationSuccess(Request $request, TokenInterface $token): ?Response
     {
-        $this->authenticatedSessionStateHandler->setCurrentRequestUri($request->getUri());
+        $redirectUri = $this->authenticatedSessionStateHandler->getCurrentRequestUri();
```

Comment:
Is no fallback needed?
It is a given that the URL is always set?

In theory it might return an empty string. Maybe fallback to '/' in that case?

Reply:
This should be always set in: `\Surfnet\StepupSelfService\SelfServiceBundle\Security\Authentication\SamlAuthenticator::start`
Or: \Surfnet\StepupSelfService\SelfServiceBundle\EventListener\ExplicitSessionTimeoutListener::checkSessionTimeout

Reply:
I howevere the fallback in the SessionStorage so this should be handled if this unexpectedly the case.